1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,31 @@
+import { fizzbuzz } from "./fizzbuzz";
 
 describe("fizzbuzz", () => {
+  it.each([3, 9, 42])("should return Fizz when given %i", (n) => {
+    expect(fizzbuzz(n)).toEqual("Fizz");
+  });
 
+  it.each([15, 45, 90])("should return Buzz when given %i", (n) => {
+    expect(fizzbuzz(n)).toEqual("FizzBuzz");
+  });
 
+  it.each([5, 10, 20])("should return Buzz when given %i", (n) => {
+    expect(fizzbuzz(n)).toEqual("Buzz");
```

Comment:
Good job with the test names (describe an example instead of a generic behavior).

Khalil later mentioned that it's good practice to also add a description of the generic behavior (using jest's `describe`) and wrap the specific examples around this description.
So it would look like:

`describe("returns 'Buzz' for multiples of five", () => {

  it("returns 'Buzz' when given 5", () => {
    expect(fizzBuzz(5)).toBe("Buzz");
  });

  it("returns 'Buzz' when given 10", () => {
    expect(fizzBuzz(10)).toBe("Buzz");
  });

});`